autogpt_platform/backend/backend/blocks/example/_api.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import annotations
+
+import json
+from typing import Any, Dict, Optional
+
+from backend.data.model import APIKeyCredentials
+from backend.util.request import Requests
```

Comment:
Comment this here that we have our own wrapper

Reply:
Why don't we use `requests` instead of the class itself.